docs/faq.md

```diff
@@ -66,3 +66,17 @@ Note that taking very frequent model snapshots will result in somewhat slower tr
 
 When saving a model snapshot, Casanovo will use the validation data to compute performance measures (training loss, validation loss, amino acid precision, and peptide precision) and print this information to the console and log file.
 After your training job is finished, you can identify the best performing model that achieves the maximum peptide and amino acid precision from the log file and use the corresponding model snapshot.
+
+**Even though I added new post-translational modifications to the configuration file, Casanovo didn't identify those peptides.**
+
+The [`config.yaml` configuration file](https://github.com/Noble-Lab/casanovo/blob/main/casanovo/config.yaml) contains all amino acids and post-translational modifications (PTMs) that Casanovo knows.
```

Comment:
Start with a simple summary answer, e.g., "Casanovo can only make predictions using PTMs that were included when training the model.  If you want to add new types of PTMs, then you will need to retrain the model."